app/views/devise/registrations/edit.html.erb

```diff
@@ -1,87 +1,92 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(@user, url: user_path(@user), method: :patch, html: { class: "my-form" }) do |form| %>
-   
-   <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name %>
-  </div>
-  
-  <div class="field">
-    <%= form.label :lastName %>
-    <%= form.text_field :lastName %>
-  </div>
-
-  <div class="field">
-    <%= form.label :birthday %>
-    <%= form.date_field :birthday %>
-  </div>
-
-  <div class="field">
-    <%= form.label :weight %>
-    <%= form.text_field :weight %>
-  </div>
-
-  <div class="field">
-    <%= form.label :height %>
-    <%= form.text_field :height %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.email_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :image_profile %>
-    <%= form.file_field :image_profile, accept: "image/jpg, image/jpeg, image/png" %>
-  </div>
+<div class="user-config">
+  <%= form_for(@user, url: user_path(@user), method: :patch, html: { class: "form" }) do |form| %>
+    <h2>Edit Profile</h2>
+
+    <div class="field">
+      <%= form.label :name %>
+      <%= form.text_field :name %>
+    </div>
+    
+    <div class="field">
+      <%= form.label :lastName %>
+      <%= form.text_field :lastName %>
+    </div>
+
+    <div class="field">
+      <%= form.label :birthday %>
+      <%= form.date_field :birthday %>
+    </div>
+
+    <div class="field">
+      <%= form.label :weight %>
+      <%= form.text_field :weight %>
+    </div>
+
+    <div class="field">
+      <%= form.label :height %>
+      <%= form.text_field :height %>
+    </div>
```

Comment:
in each field add a `required: true` to have a end to end validation.

Reply:
done :+1: